tests/dashbio_demos/app_clustergram.py

```diff
@@ -285,62 +306,40 @@ def layout():
 
                         html.Hr(),
 
-                        html.Div(
-                            'Add or remove all group markers:',
-                            className='clustergram-option-name'
-                        ),
-
-                        html.Br(),
-
                         html.Div(
                             id='add-group-markers',
-                            title='Annotate your heatmap by labelling clusters; ' +
-                            'hover over the clusters on the dendrogram to get their ' +
-                            'index.',
                             children=[
-                                dcc.Dropdown(
-                                    id='row-or-col-group',
-                                    options=[
-                                        {'label': 'Row group', 'value': 'row'},
-                                        {'label': 'Column group', 'value': 'col'}
-                                    ]
+                                html.Div(
+                                    className='clustergram-option-name',
+                                    children='Add annotations'
                                 ),
-                                dcc.Input(
-                                    id='group-number',
-                                    placeholder='group number',
-                                    type='number',
-                                    value=''
+                                html.Button(
+                                    id='remove-all-group-markers',
+                                    children='Remove all',
+                                    n_clicks=0,
+                                    n_clicks_timestamp=0
                                 ),
-                                dcc.Input(
-                                    id='color',
-                                    placeholder='color',
-                                    type='text',
-                                    value=''
+                                html.Br(),
+                                html.Div(className='clustergram-option-desc', children=[
+                                    'Annotate your heatmap by labelling clusters; '
+                                    'choose a color for the annotation, as well as '
+                                    'text for the annotation, below. Then, click '
```

Comment:
Hmm, I would move 'below' to "choose below a color ..."

Reply:
I actually find the way that I have it clearer -- is it conventional to structure the sentence in the way you've suggested?

Reply:
I think the more conventional way would be to write "below, choose a color" (look it up, but I've never seen 'below' isolated at the very end of a sentence, with that separating comma: ", below.").

Reply:
Changed in cd7b7b0!